src/components/CreateProfile/WorkInformation/WorkInformation.tsx

```diff
@@ -59,35 +58,49 @@ const WorkInformation = () => {
       </div>
 
       <div className={styles.right}>
-        <InputFormError error={touched.position && errors.position}>
-          <DropdownBio
-            id="position"
+        <InputFormError
+          error={
+            touched[WorkInformationFormKeys.POSITION] &&
+            errors[WorkInformationFormKeys.POSITION]?.value
+          }
+        >
+          <DropdownSelect
+            id={WorkInformationFormKeys.POSITION}
             label="Position"
             text="Choose position"
-            options={filterLists.position}
-            selectedValue={values.position}
-            name="position"
+            options={mappedSeniorityLevel}
+            selectedValue={values[WorkInformationFormKeys.POSITION]}
+            name={WorkInformationFormKeys.POSITION}
           />
         </InputFormError>
-        <InputFormError error={touched.seniority && errors.seniority}>
-          <DropdownBio
-            id="seniority"
+        <InputFormError
+          error={
+            touched[WorkInformationFormKeys.SENIORITY] &&
+            errors[WorkInformationFormKeys.SENIORITY]?.value
+          }
+        >
+          <DropdownSelect
+            id={WorkInformationFormKeys.SENIORITY}
             label="Seniority"
             text="Choose seniority"
-            options={filterLists.seniority}
-            selectedValue={values.seniority}
-            name="seniority"
+            options={mappedSeniorityLevel}
+            selectedValue={values[WorkInformationFormKeys.SENIORITY]}
+            name={WorkInformationFormKeys.SENIORITY}
           />
         </InputFormError>
-        <InputFormError error={touched.techStack && errors.techStack}>
+        <InputFormError
+          /* TODO: Fix it */
+          // error={
+          //   touched[WorkInformationFormKeys.TECH_STACK] &&
+          //   errors[WorkInformationFormKeys.TECH_STACK]
+          // }
```

Comment:
I think the error is connected with the Yup schema. We have an array of strings there but in fact, we have an array of objects `DropdownOption`. We should change the techStack validation schema